recipes-kernel/linux/files/0001-Backport-BCM270X_DT-Rename-Pi-Zero-W-DT-files.patch

```diff
@@ -0,0 +1,65 @@
+From b3fa15a3e7b5229bd920be264499c86933ae3d37 Mon Sep 17 00:00:00 2001
+From: Phil Elwell <phil@raspberrypi.org>
+Date: Tue, 28 May 2019 16:36:04 +0100
+Subject: [PATCH] Backport BCM270X_DT: Rename Pi Zero W DT files
+
+This patch needed to be backported to series 4.14.y because it is also supported in meta-raspberrypi for Yocto Project.
+Unfortunately, the rename affected the correct building of the 4.14 series kernel, which now results broken.
+Applying the same renames on the 4.14.y series would fix the issue for those still using that kernel version.
+
+The downtream Pi Zero W dts file uses the digit 0, whereas upstream
+chose to spell it out - "zero-w". The firmware has, for a long time,
+looked for bcm2708-rpi-zero-w.dtb first before falling back to the
+numerical version. Therefore it is better to follow upstream and
+make the switch to "bcm2708-rpi-zero-w".
+
+At the same time, remove some overrides that duplicate values
+inherited from the shared .dtsi files.
+
+Signed-off-by: Phil Elwell <phil@raspberrypi.org>
+Signed-off-by: Francesco Giancane <francescogiancane8@gmail.com>
```

Comment:
This misses upstream-status but also the commit log needs tweaking, Please take a look to [these](https://meta-raspberrypi.readthedocs.io/en/latest/contributing.html) guidelines and also `git log`  will give you examples.

Reply:
@agherzan ,
Ok, thanks for pointing me out those missing things. At the moment the patch has been already merged upstream in raspberrypi/linux rpi-4.14.y, do you think is better to just rework my PR to build the latest kernel? (thus including this fixes?)

Reply:
That is great. Let's go with the update.

Reply:
Hi @agherzan ,

PR updated with the latest 4.14.y tip including the patch. Done some commit description modifications and removed the included patch as it is now mainline.

Reply:
@fgiancane8 Can you also make sure the line length has a 78 chars limit as well? In the commit log. Thanks.

Reply:
@agherzan ,
reviewed the PR name and also all the cosmetic changes to line length in commit log applied.

Thanks,
F

Reply:
Thanks @fgiancane8 . Very nice.